currency.go

```diff
@@ -179,11 +180,13 @@ var currencies = Currencies{
 	SGD: {Decimal: ".", Thousand: ",", Code: SGD, Fraction: 2, NumericCode: "702", Grapheme: "$", Template: "$1"},
 	SHP: {Decimal: ".", Thousand: ",", Code: SHP, Fraction: 2, NumericCode: "654", Grapheme: "\u00a3", Template: "$1"},
 	SKK: {Decimal: ".", Thousand: ",", Code: SKK, Fraction: 2, NumericCode: "", Grapheme: "Sk", Template: "$1"},
+	SLE: {Decimal: ".", Thousand: ",", Code: SLE, Fraction: 2, NumericCode: "694", Grapheme: "Le", Template: "1 $"},
```

Comment:
NumericCode is `925`

![CleanShot 2024-04-02 at 10 20 38@2x](https://github.com/Rhymond/go-money/assets/3577968/09ac3bd0-e533-42c2-a6dc-a8fdf9b6b3ac)

Reply:
Just updated it. It seems that the iban.com still shows the SLE (old Leone)